spec/fixtures/json/sam-s-simple-theme.dcstyle.json

```diff
@@ -1 +1 @@
f={{topic.url}}\u003e{{format-date topic.createdAt format=\"tiny\"}}\u003c/a\u003e\n  {{~/if ~}}\n   {{raw \"list/action-list\" topic=topic postNumbers=topic.liked_post_numbers className=\"likes\" icon=\"heart\"}}\n   \u003c/div\u003e\n\u003c/td\u003e\n\n\n\n{{#if controller.showLikes}}\n\u003ctd class=\"num likes\"\u003e\n  {{number topic.like_count}} \u003ci class='fa fa-heart'\u003e\u003c/i\u003e\n\u003c/td\u003e\n{{/if}}\n   \n{{#if controller.showOpLikes}}\n\u003ctd class=\"num likes\"\u003e\n  {{number topic.op_like_count}} \u003ci class='fa fa-heart'\u003e\u003c/i\u003e\n\u003c/td\u003e\n{{/if}}\n\n{{raw \"list/posts-count-column\" topic=topic}}\n\n\u003ctd class=\"last-post\"\u003e\n\u003cdiv class='poster-avatar'\u003e\n\u003ca href=\"{{topic.lastPostUr}}\" data-user-card=\"{{topic.last_poster_username}}\"\u003e{{avatar topic.lastPoster usernamePath=\"username\" imageSize=\"medium\"}}\u003c/a\u003e\n\u003c/div\u003e\n\u003cdiv class='poster-info'\u003e\n\u003ca href=\"{{topic.lastPostUrl}}\"\u003e\n{{format-date topic.bumpedAt format=\"tiny\"}}\n\u003c/a\u003e\n\u003cspan class='editor'\u003e\u003ca href=\"/users/{{topic.last_poster_username}}\" data-auto-route=\"true\" data-user-card=\"{{topic.last_poster_username}}\"\u003e{{topic.last_poster_username}}\u003c/a\u003e\u003c/span\u003e\n\u003c/div\u003e\n\u003c/td\u003e\n\u003c/script\u003e\n\n\u003cscript type='text/x-handlebars' data-template-name='topic-list-header.raw'\u003e\n  {{raw \"topic-list-header-column\" order='posts' name='topic.title'}}\n  \n  {{#if showLikes}}\n     {{raw \"topic-list-header-column\" sortable='true' order='likes' number='true' forceName='Likes'}}\n  {{/if}}\n  {{#if showOpLikes}}\n     {{raw \"topic-list-header-column\" sortable='true' order='op_likes' number='true' forceName='Likes'}}\n  {{/if}}\n  {{raw \"topic-list-header-column\" sortable='true' number='true' order='posts' forceName='Replies'}}\n  {{raw \"topic-list-header-column\" sortable='true' order='activity' forceName='Last Post'}}\n\u003c/script\u003e\n\n\u003cscript\u003e\n\n(function(){\n\nvar TopicListItemView = require('discourse/components/topic-list-item').default;\n\n\nTopicListItemView.reopen({\n    showCategory: function(){\n        return !this.get('controller.hideCategory') \u0026\u0026\n                this.get('topic.creator') \u0026\u0026\n                this.get('topic.category.name') !== 'uncategorized';\n    }.property()\n});\n   \n})();\n \n\u003c/script\u003e\n"}],"child_themes":[],"remote_theme":{"id":1,"remote_url":"https://github.com/SamSaffron/discourse-simple-theme.git","remote_version":"49986016d76be1f526d5dc44704e6701021db900","local_version":"49986016d76be1f526d5dc44704e6701021db900","about_url":"https://meta.discourse.org/t/sams-personal-minimal-topic-list-design/23552","license_url":"https://github.com/SamSaffron/discourse-simple-theme/blob/master/LICENSE.txt","commits_behind":0,"remote_updated_at":"2017-05-08T06:17:14.487Z","updated_at":"2017-05-08T06:17:14.501Z"}}}
```

Comment:
What has even changed here haha?

Reply:
A key in the serializer payload has changed xD